controls/ubuntu-18.04-server-cis-5.2.3.rb

```diff
@@ -58,14 +58,18 @@
   "
   impact 0.5
   tag severity: "medium"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
   tag nist: ["AC-3 (3)", "Rev_4"]
   tag cis_level: 1
   tag cis_controls: ["14.6", "Rev_7"]
   tag cis_rid: "5.2.3"
+  if (File.exist?('/etc/ssh'))
+    Array(Dir["/etc/ssh/ssh_host_*_key.pub"]).each do |key_file|
+      describe file(key_file) do
+        it { should be_file }
+        it { should be_owned_by 'root' }
+        it { should be_grouped_into 'root' }
+        its('mode') { should cmp '00644' }
```

Comment:
same as above

Reply:
note, I did not run this code yet but I think it gives you the idea, we don't want to have 'uncovered caes' or 'false positives or negities' or even worse - no data - in our tests.

Reply:
And if we do have 'no data' - aka a `skip` - then we should always 'document' - aka describe statment - as to why and what the user should do.

Reply:
Okay, I think I think I see the rationale here. I brought up #8 asking about this. My thinking was that if the control said that all SSH public keys needed to have the right ownership, etc, and you had no such files (perhaps because ssh was not used at all) then the control was satisfied, and should appear as a 'pass'. But if I understand you correctly, it is preferable to mark such as case as 'skip' to ensure that a human reviews it.